src/virt.c

```diff
@@ -31,6 +31,8 @@
 #include <libxml/parser.h>
 #include <libxml/tree.h>
 #include <libxml/xpath.h>
+#include <libgen.h> /* for basename(3) */
+#include <stdbool.h>
```

Comment:
Please don't use `<stdbool.h>`, it is causing problems on some systems and/or in combination with some libraries …